tests/packaged_modules/test_autofolder.py

```diff
@@ -0,0 +1,439 @@
+import importlib
```

Comment:
It would be great if we could avoid test duplication, so perhaps we can have this file for the main "folder loader" tests and `test_imagefolder` and `test_audiofolder` for the ones specific to the loader (but it's ok to address this in another PR)